### DISCUSSION TITLE
Ensuring Updates for Dependent User Stories when Features are Postponed

### DISCUSSION THREAD

**Discussion by TuncAlt** [neuefische/questions#541] [Web] [answered]
How can you ensure that all dependent user stories are updated when a user story is not fully completed and a feature is postponed?

In my case, the LineChart still needs to be added to my DevicePage, and I would like to postpone it until the future. However, I need the DevicePage to access the Edit and Setting functions. I would like to partially complete this UserStory and begin working on the other UserStories to implement the Edit and Setting functions.

**Comment by F-Kirchhoff** [comment, marked as answer]
Hey,
my suggestion is that you create a new user story for adding the Line Chart to the Device Page and move all parts from your current user story about that Chart to that new user story. Then you can finish your current story and start working on the next ones.

**Reply by TuncAlt** [reply]
Perfect! Thank you.
